tests/real/enterprise.rs

```diff
@@ -588,7 +588,7 @@ fn test_batch_of_jobs_can_be_initiated() {
 
     assert!(b.add(job_1).unwrap().is_none());
     assert!(b.add(job_2).unwrap().is_none());
-    assert_eq!(b.add(job_3).unwrap().unwrap(), "check-check");
+    assert_eq!(b.add(job_3).unwrap().unwrap(), Value::from("check-check"));
```

Comment:
I _believe_ `serde_json::Value` implements `PartialEq<str>`, so the `Value::from` here _may_ not be needed.

Reply:
Indeed!